src/TestableIO.System.IO.Abstractions.TestingHelpers/MockFileSystem.cs

```diff
@@ -243,6 +243,12 @@ public void AddFile(string path, MockFileData mockFile)
             }
 
             var directoryPath = Path.GetDirectoryName(fixedPath);
+            if (directoryPath == null)
```

Comment:
I think, it is easier to understand, if you combine the two if-conditions, e.g.:
```csharp
            if (directoryPath == null)
            {
                AddDrive(fixedPath, new MockDriveData());
            }
            else if (!DirectoryExistsWithoutFixingPath(directoryPath))
            {
                // ...
            }
```

Reply:
sure, fixed it